### ISSUE TITLE
URL regex matching not working

### ISSUE THREAD

**Issue by luispabon** [stylish-userstyles/stylish#314]
`URLs matching the regex` option doesn't seem to be working. For instance, adding `jenkins` doesn't match the url `https://jenkins`.

I've tried explicitly adding the slash delimiters, with and without case sensitive modifier...

**Comment by myfonj** [comment]
Regex must match **entire** URL, not just part.  **`.*jekins.*`** should work.

Btw, good to know, the are (technically) no URLs without slash after domain, despite they are often displayed this way.

**Comment by luispabon** [comment]
Thank you, `.*jenkins.*` does the trick for me. Probs worth mentioning on Stylish's UI that the whole URL, including the protocol, need matching. The current wording is misleading, as `jenkins` indeed matches `http://jenkins` hence my confusion.

**Comment by myfonj** [comment]
Yup. Or perhaps make attractive link to the docs [1], which BTW mentions that

> regexp - for advanced matching with wildcards (**including the protocol**) 

and also

> The regular expression you write must match the **entirety of the URL**. This means that using `^` and `$` (to match the beginning and end of the string) are unnecessary.

(emphasis mine).

[1] https://github.com/stylish-userstyles/stylish/wiki/Applying-styles-to-specific-sites

**Comment by luispabon** [comment]
A link to the docs would do nicely I reckon :+1:
